### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by blackdwarf**
Update README.md with some more info.

skip ci please

/cc @richlander @piotrpMSFT

### CODE REVIEW

**Comment by eerhardt** [comment]
I'm not expecting you to do this, @blackdwarf, but it would be nice if we had a build matrix in the README similar to dotnet/cli.

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# Core-Setup
\ No newline at end of file
+.NET Core Runtime & host setup repo
+===================================
+
+This repo contains the code to build the .NET Core runtime, libraries and shared host (`dotnet`) installers for 
+all supported platforms. It **does not** contain the actual sources to .NET Core runtime; this source is split across 
+the dotnet/corecrl repo (runtime) and dotnet/corefx repo (libraries).
```
Comment (eerhardt) [inline README.md]: type-o `dotnet/corecrl`

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+document in the dotnet/cli repo. That is the first step to get acquantied with the overall plan and experience we have
+thought up for installing .NET Core bits. 
+
+## Filing issues
+This repo should contain issues that are tied to the installation of the "muxer" (the `dotnet` binary) and installation 
+of the .NET Core runtime and libraries. 
+
+For other issues, please use the following repos:
+
+- For overall .NET Core SDK issues, file on dotnet/cli repo
```
Comment (eerhardt) [inline README.md]: nit: links?

**Comment by blackdwarf** [comment]
Fixed. Merging.

**Pull request merged**